packages/astro/src/core/build/index.ts

```diff
@@ -157,9 +158,10 @@ class AstroBuilder {
 		await runHookBuildStart({ config: this.settings.config, logging: this.logger });
 		this.validateConfig();
 
-		this.logger.info('build', `output target: ${colors.green(this.settings.config.output)}`);
+		this.logger.info('build', `output: ${blue('"' + this.settings.config.output + '"')}`);
+		this.logger.info('build', `outDir: ${blue(fileURLToPath(this.settings.config.outDir))}`);
 		if (this.settings.adapter) {
-			this.logger.info('build', `deploy adapter: ${colors.green(this.settings.adapter.name)}`);
+		  this.logger.info('build', `adapter: ${blue('@vercel/serverless')}`);
```

Comment:
Presumably this is supposed to be the adapter the user is using.

Reply:
that's what I get for trying to fix this quickly from the web UI on my phone... thanks for the thorough review on this @Princesseuh